docpages/example_programs/misc/using-emojis.md

```diff
@@ -0,0 +1,29 @@
+\page using-emojis Using Emojis
+
+Need your bot to use an emoji? Then you've come to the right place! Here are three examples of using emojis.
+
+\note If your bot isn't in the guild where you want to use the custom emoji from, it won't work, giving you dpp::exception_error_code::err_unknown_emoji.
```

Comment:
Figured it out, change `dpp::exception_error_code::err_unknown_emoji` to `dpp::err_unknown_emoji` (without codeblock ofc). Not sure why doxygen isn't accepting it with the current state but oh well.